tox.ini

```diff
@@ -118,6 +118,7 @@ setenv =
 install_command =
     python -I -m pip install {opts} {packages}
 commands =
+    ; tests for docs fail with parsel < 1.8.0
```

Comment:
Isn’t this a bit out of place here?

Reply:
The context is #5902

Reply:
My bad, for some reason I thought this message was about freezing parsel for the tests, I did not realize it was about removing the docs from them. :facepalm: